cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
The whole point of being able to use this include functionality is so that you *dont* have to make a script to combine all the files into one. I definitely think the goal here isn't getting communicated.